deployments/fpga_admissionwebhook/af-crd.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: acceleratorfunctions.fpga.intel.com
+spec:
+  group: fpga.intel.com
+  version: v1
+  scope: Namespaced
+  names:
+    plural: acceleratorfunctions
+    singular: acceleratorfunction
+    kind: AcceleratorFunction
+    shortNames:
+    - af
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            interfaceId:
+              type: string
+              pattern: '^[0-9a-f]+$'
```

Comment:
maybe better to declare expected length ?

Reply:
is this length guaranteed by a spec?